.github/workflows/shell-app-android.yml

```diff
@@ -0,0 +1,71 @@
+name: Android Shell App
+
+on:
+  workflow_dispatch:
```

Comment:
Love this! ❤️

Reply:
You'll love it even more 😊 https://github.com/expo/expo/pull/9732